vllm/v1/core/kv_cache_manager.py

```diff
@@ -272,8 +273,9 @@ def allocate_slots(
         # The number of computed tokens is the number of computed tokens plus
         # the new prefix caching hits
         num_computed_tokens = request.num_computed_tokens + num_new_computed_tokens
+        num_tokens_target_model = num_computed_tokens + num_new_tokens
```

Comment:
add comment.
"main model" may be a better name?
do we need min(max_model_len) here?